### ISSUE TITLE
Tongass National Forest polygon needs to be simplified

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/geospatial-vector-veracity#104]
https://earthmaps.io/boundary/area/USFS21

This polygon is valid, but the rendered GeoJSON is way too big (35MB) and it won't load in the browser.

We need a simplified, but still valid, representation of this place.

**Comment by charparr** [comment]
A 100 m simplification yields a 7.2 MB GeoJSON and still looks decent given how crenulated that part of the AK coast is. In this screenshot example, the purple is the original, and the yellow is the simplified overlain on top. The value of 100 means that all parts of a simplified geometry will be no more than 100 m distance from the original. 
![image](https://user-images.githubusercontent.com/7774271/193197061-118def2d-9f6c-487a-b50b-8fcc39b47f38.png)

**Comment by charparr** [comment]
Yeah, this polygon still does not render in NCR probably because it is too large so this ticket is still relevant.
